server/src/language/parser.js

```diff
@@ -778,6 +780,31 @@ export default class Parser {
             }
             break;
 
+          case `EXEC`:
+            if (parts[1] === `SQL`) {
+              const indexDeclare = parts.findIndex(el => el === `DECLARE`);
+              const indexCursor = parts.findIndex(el => el === `CURSOR`);
+              let indexCursorName = 0;
+              if (indexCursor-1 == indexDeclare+1) {
+                indexCursorName = indexCursor-1;
+              }
+              if (currentCursor === undefined 
+               && indexDeclare >=0 
```

Comment:
Indentation here is incorrect. eslint should be able to fix this up automatically for you. (or Format Document in VS Code)